### PR TITLE
Update docs on funnelcakes, analytics, and a/b tests

### PR DESCRIPTION
**Pull request by stephaniehobson**

### CODE REVIEW

File: docs/abtest.rst [hunk 1]
```diff
@@ -54,9 +54,9 @@ different.
 Create the variant page like you would a new page. Make sure it is ``noindex``
 and does not have a ``canonical`` url.
 
-```
-{% block canonical_urls %}<meta name="robots" content="noindex,follow">{% endblock %}
-```
+.. code-block:: python
```
Comment (pmac) [inline docs/abtest.rst]: should be `.. code-block:: jinja` and have a blank line before the content.

File: docs/abtest.rst [hunk 2]
```diff
@@ -68,13 +68,12 @@ Create a .js file where you initialize Traffic Cop and include that in the
...
 extra js include in a `switch <http://bedrock.readthedocs.io/en/latest/install.html#feature-flipping-aka-switches>`_.
 
-```
-{% block experiments %}
-  {% if switch('experiment-berlin-video', ['de']) %}
-    {{ js_bundle('firefox_new_berlin_experiment') }}
-  {% endif %}
-{% endblock %}
-```
+.. code-block:: javascript
```
Comment (pmac) [inline docs/abtest.rst]: This is also `jinja`.

File: docs/funnelcake.rst [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+=================
+
+Funnel cakes
+------------
+
+In addition to being `an American delicacy <https://en.wikipedia.org/wiki/Funnel_cake>`
+funnel cakes are what we call special builds of Firefox. They can come with
+extensions preinstalled and/or a custom first-run experience.
+
+    "The whole funnelcake system is so marred by history at this point I don't
```
Comment (pmac) [inline docs/funnelcake.rst]: LOL 🤣

File: docs/funnelcake.rst [hunk 4]
```diff
@@ -0,0 +1,92 @@
...
+Currently bedrock only supports funnelcakes for "stub installer platforms". Which
+means they are windows only. However, funnelcakes can be made for all platforms
+so `bedrock support may expand <https://github.com/mozilla/bedrock/issues/6251>`_.
+
+We signal to bedrock that we want a funnelcake when linking to the download
+page by appending the query variable `f` with a value equal to the funnelcake
+number being requested.
+
+.. code-block::
+    https://bedrock-demo-shobson.oregon-b.moz.works/en-US/firefox/download/thanks/?f=137
```
Comment (pmac) [inline docs/funnelcake.rst]: Can probably just use `www.mozilla.org` domain here instead of your demo.

File: docs/funnelcake.rst [hunk 5]
```diff
@@ -0,0 +1,92 @@
...
+
+- Does it have executive approval?
+- Did legal sign off?
+- Has it had a security review?
+
+On the technical side:
+
+- Is the switch enabled?
+- Is the variable being passed?
+- If `force_direct` is set to True on the download button macro the funnelcake
```
Comment (pmac) [inline docs/funnelcake.rst]: I don't think that's true actually. I believe it's when `force_direct=True` (like in the scene2 templates) that you get the bouncer link to the `firefox-stub-f135` products. Without `force_direct` the button links to the `/download/thanks/` page and passes the `f` parameter along.

File: docs/analytics.rst [hunk 6]
```diff
@@ -118,3 +118,37 @@ When a banner is dismissed:
         'data-banner-dismissal': '1',
         'event': 'in-page-interaction'
     });
+
+
+When doing a/b tests configure something like the following.
+
+.. code-block:: javascript
```
Comment (pmac) [inline docs/analytics.rst]: These aren't working. The blank line after the `code-block` declaration is required.

File: docs/analytics.rst [hunk 7]
```diff
@@ -118,3 +118,37 @@ When a banner is dismissed:
...
+    }
+
+
+Some notes on how this looks in GA
+----------------------------------
+
+``data-link-type="button"`` and ``data-link-name=""`` trigger a generic link
+click with the following structure:
+
+    Event Category: {{page ID}} Interactions
```
Comment (pmac) [inline docs/analytics.rst]: You need a [line block](http://docutils.sourceforge.net/docs/ref/rst/restructuredtext.html#line-blocks) here. e.g.:

```rst
| Event Category: {{page ID}} Interactions
| Event Action: {{data-link-type}} click
| Event Label: {{data-link-name}}
```

That will preserve the linebreaks.

File: docs/funnelcake.rst [hunk 8]
```diff
@@ -0,0 +1,92 @@
...
+page by appending the query variable `f` with a value equal to the funnelcake
+number being requested.
+
+.. code-block::
+    https://bedrock-demo-shobson.oregon-b.moz.works/en-US/firefox/download/thanks/?f=137
+
+Bedrock checks to see if the funnelcake is configured (this is handled in the
+`www-config repo <https://github.com/mozmeao/www-config/blob/master/waffle_configs/bedrock-prod.env>`_)
+
+.. code-block::
```
Comment (pmac) [inline docs/funnelcake.rst]: You can call this `.. code-block:: bash` I think. It's basically shell syntax.

File: docs/funnelcake.rst [hunk 9]
```diff
@@ -0,0 +1,92 @@
...
+craft your own download button:
+
+.. code-block:: html
+    <a href="https://download.mozilla.org/?product=firefox-election-edition&os=win&lang=en-US">
+    Download</a>
+
+------------
+
+Bugs that might have useful info:
+- https://bugzilla.mozilla.org/show_bug.cgi?id=1450463
```
Comment (pmac) [inline docs/funnelcake.rst]: Need an empty line above the list so that it'll render as a list. The one below as well.

File: docs/abtest.rst (hunk 1 above)
Comment (stephaniehobson) [reply docs/abtest.rst]: Significant white space is my nemesis.

**Comment by stephaniehobson** [comment]
Updated. Thanks for the tip on seeing them rendered on GitHub.

File: docs/abtest.rst [hunk 10]
```diff
@@ -119,9 +120,10 @@ Some things to consider checking:
 
 A/B Test PRs that might have useful code to reuse
 ---
```
[the remark this answers is not in the document]
Comment (stephaniehobson) [reply docs/abtest.rst]: I don't know why anyone things this markup stuff is easier than HTML.

Updated.

**Pull request merged**